image_animation.py

```diff
@@ -29,16 +29,15 @@
 source_image = imageio.imread(source_path)
 source_image = resize(source_image,(256,256))[..., :3]
 
-generator, kp_detector = load_checkpoints(config_path='config/vox-256.yaml', checkpoint_path=checkpoint_path)
+relative=True
+adapt_movement_scale=True
+cpu=True
```

Comment:
if torch.cuda.is_available():
    cpu = False
else:
    cpu = True

would be better than plainly setting it to cpu